Dockerfile

```diff
@@ -1,13 +1,16 @@
 # Start with a Golang-enabled base image.
 FROM golang:1.10.0 as builder
 
-# Fetch the CoreDNS repo.
+# Fetch all dependencies.
 RUN go get github.com/coredns/coredns
 RUN go get github.com/opentracing/opentracing-go
+RUN go get github.com/sirupsen/logrus
```

Comment:
remind me why these are not vendored?

Reply:
The entire coredns repo is the only piece that's annoying to vendor